test/test_workbook.py

```diff
@@ -470,8 +470,11 @@ def test_publish_with_hidden_view(self):
                                                          hidden_views=['GDP per capita'])
 
             request_body = m._adapter.request_history[0]._request.body
-            self.assertTrue(re.search(rb'<views><view.*?hidden=\"true\".*?\/><\/views>', request_body))
-            self.assertTrue(re.search(rb'<views><view.*?name=\"GDP per capita\".*?\/><\/views>', request_body))
+            # order of attributes in xml is unspecified
```

Comment:
I don't think this change clearly communicates what is being tested. In the previously written tests, it reads as "this attribute exists in the response". In your change it reads as "the response is exactly this."  What happens if the views as a third property added, now you need 9 variations of the check. If you don't want to use regexes, then parse the xml and validate that the attributes hidden and name exist. I don't feel like the current change is an improvement in test strategy.

Reply:
Yea in 12aacb9c7e1bac0a9c76b1e01ef3bb55548e77fa I switched to the dual regex checks here specifically because in Python 3.8 the order of attributes changed.

Reply:
This isn't intended as a change in strategy, this was just to make the tests pass. The previous test failed because the xml was reordered on my machine.